sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/component.py

```diff
@@ -14,7 +14,10 @@
     ComponentContainer,
     ComponentContainerProperties,
     ComponentVersion,
-    ComponentVersionProperties,
+)
+from ..._restclient.v2023_08_01_preview.models import (
```

Comment:
sorry, 1 more comment, I think you can directly change the import in line 13, use `from .v2023_08_01_preview.models import` to replace `from .v2022_10_01.models`. The same to pipeline_job, we have updated all related references.

Reply:
I will try it, but I will revert code if tests run fail.
I modify 18 line because of `to_dict_from_reset` will fail.